content/docs/installation/kubectl.md

```diff
@@ -88,3 +97,23 @@ experiencing issues then run:
 ```bash
 $ kubectl delete apiservice v1beta1.webhook.cert-manager.io
 ```
+
+#### Deleting pending challenges
+
+`Challenge`s can get stuck in a pending state when the finalizer is unable to complete 
+and Kubernetes is waiting for the cert-manager controller to finish. 
+This happens when the controller is no longer running to remove the flag, 
+and the resources are defined as needing to wait.
+You can fix this problem by doing what the controller does manually. 
+
+First, delete existing cert-manager webhook configurations, if any:
+
+```bash
+$ kubectl delete validatingwebhookconfigurations cert-manager-webhook
+$ kubectl delete validatingwebhookconfigurations cert-manager-webhook
```

Comment:
I think this should be `kubectl delete mutatingwebhookconfigurations cert-manager-webhook`